src/applications/personalization/profile/components/personal-information/ContactInformationView.jsx

```diff
@@ -31,7 +33,7 @@ const ContactInformationView = props => {
         />
 
         {fieldName === FIELD_NAMES.MOBILE_PHONE && (
-          <ReceiveTextMessages fieldName={FIELD_NAMES.MOBILE_PHONE} />
```

Comment:
Sadly I cannot also delete the code for this `ReceiveTextMessages` component because it's used in a legacy `MobilePhone` component.